patches/squirrel.mac/fix_use_non-deprecated_openappliationaturl.patch

```diff
@@ -0,0 +1,54 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Charles Kerr <charles@charleskerr.com>
+Date: Mon, 29 Jul 2024 18:12:40 -0500
+Subject: refactor: use non-deprecated openAppliationAtURL
+
+Use openAppliationAtURL instead of launchApplicationAtURL,
+which was deprecated in macOS 11.
+
+This patch can be removed if/when upstream removes its use of
+deprecated macOS API.
+
+diff --git a/Squirrel/ShipIt-main.m b/Squirrel/ShipIt-main.m
+index 671f8fa2104df85046ff813d4b824a65caae502f..d9efbe5f8337921901509d00fa022b3c41489bf6 100644
+--- a/Squirrel/ShipIt-main.m
++++ b/Squirrel/ShipIt-main.m
+@@ -186,12 +186,8 @@ static void installRequest(RACSignal *readRequestSignal, NSString *applicationId
+ 							NSLog(@"New ShipIt exited");
+ 						} else {
+ 							NSLog(@"Attempting to launch app on lower than 11.0");
+-							if (![NSWorkspace.sharedWorkspace launchApplicationAtURL:bundleURL options:NSWorkspaceLaunchDefault configuration:@{} error:&error]) {
```

Comment:
it looks like this deprecated code was deleted instead of being updated to use non-deprecated `openApplicationAtURL:configuration:completionHandler:`?